practice-two/src/js/script.js

```diff
@@ -1 +1,144 @@
+// Query elements
+const fullNameInput = document.getElementById('fullName');
+const emailInput = document.getElementById('email');
+const salaryInput = document.getElementById('salary');
+const cityInput = document.getElementById('city');
 
+// Regex for validating a value/text format
+const REGEX = {
+  alphabetRegex: /^[a-zA-Z]+ [a-zA-Z]+$/,
+  emailRegex: /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/
+}
+
+// Messages
+const MESSAGES = {
+  empty: 'Empty',
+  wrongFormat: 'Enter the wrong format. Please re-enter', 
+};
+
+const EmptyText = '';
+
+/**
+ * Display error message when user enters wrong input
+ * 
+ * @param {*} input Input element
```

Comment:
It should be HtmlInputElement. Check all type of element [here](https://developer.mozilla.org/en-US/docs/Web/API#interfaces)